docs/conf.py

```diff
@@ -108,19 +108,30 @@
 # If true, keep warnings as "system message" paragraphs in the built documents.
 #keep_warnings = False
 
+autodoc_typehints = 'none'
 
 # -- Options for HTML output ---------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'bootstrap-ska'
+html_baseurl = "https://sot.github.org/kadi/"
```

Comment:
I assume that this would also need to be github.io to do anything?

Reply:
Oops. Yup.

Reply:
Sure, I thought homepage_url was used to have an absolute link for the "home" style links in the current setup but unclear if this html_baseurl is used with your experimental setup.

Reply:
the theme might use that internally to provide links (lie on the top banner).